/**:
  ros__parameters:
    command_topic: "ctrl_cmd"
    state_topic: "vehicle_kinematic_state"
    trajectory_topic: "trajectory"
    tf_topic: "tf"
    static_tf_topic: "tf_static"
    diagnostic_topic: "control_diagnostic"
    debug_trajectory_publish_period_ms: 100  # if 0 or missing, no publishing happens
    vehicle:
      cg_to_front_m: 1.2
      cg_to_rear_m: 1.5
      cg_to_thrusters_long_m: 1.0
      cg_to_thrusters_lateral_m: 0.6
      front_corner_stiffness: 17000.0
      rear_corner_stiffness: 20000.0
      mass11_kg: 1460.0
      mass22_kg: 1460.0
      mass33_kgm2: 1460.0
      damping11_kgm_s: 2170.0
      damping22_kgm_s: 2170.0
      damping33_kgm2_s: 2170.0
      width_m: 2.0
    controller:
      interpolation: true
      sample_tolerance_ms: 20
      control_lookahead_ms: 100
      limits:
        min_longitudinal_velocity_mps: 0.01
        max_longitudinal_velocity_mps: 35.0
        min_lateral_velocity_mps: -3.0
        max_lateral_velocity_mps: 3.0
        min_acceleration_mps2: -3.0
        max_acceleration_mps2: 3.0
        min_yaw_rate_rps: -3.0
        max_yaw_rate_rps: 3.0
        min_jerk_mps3: -10.0
        max_jerk_mps3: 10.0
        min_thruster_force_newton: -200
        max_thruster_force_newton: 200
      behavior:
        stop_rate_mps2: 3.0
        time_step_ms: 100
        is_temporal_reference: true
      weights:
        nominal:
          pose_x: 10.0
          pose_y: 10.0
          heading: 10.0
          longitudinal_velocity: 10.0
          lateral_velocity: 10.0
          yaw_rate: 10.0
          acceleration: 1.0
          jerk: 10.0
          left_cmd: 30.0
          right_cmd: 10.0
        terminal:
          pose_x: 1000.0
          pose_y: 1000.0
          heading: 1000.0
          longitudinal_velocity: 1000.0
          lateral_velocity: 1000.0
          yaw_rate: 1000.0
          acceleration: 1000.0
          jerk: 1000.0
          left_cmd: 1000.0
          right_cmd: 1000.0
